### ISSUE TITLE
Test suite does not pass, even when comparing the same commit

### ISSUE THREAD

**Issue by conradtchan** [cpinte/mcfost#65]
Using commit dc5250a157a48a63fbea11f7822888130db64198 because db94244c41cad9c1517363b1607f8596af1c55c0 has an incompatible utils version.

Following the procedure described here: https://mcfost.readthedocs.io/en/latest/test_suite.html

Generating the reference data and testing against itself results in some test failures.

```
test_mcfost.py::test_mcfost_bin PASSED                                                                          [  2%]
test_mcfost.py::test_Temperature[discF_00500] PASSED                                                            [  4%]
test_mcfost.py::test_Temperature[ref3.0_multi] PASSED                                                           [  6%]
test_mcfost.py::test_Temperature[debris] PASSED                                                                 [  8%]
test_mcfost.py::test_Temperature[ref3.0] PASSED                                                                 [ 10%]
test_mcfost.py::test_SED[discF_00500] SKIPPED (No SED)                                                          [ 12%]
test_mcfost.py::test_SED[ref3.0_multi] PASSED                                                                   [ 14%]
test_mcfost.py::test_SED[debris] PASSED                                                                         [ 16%]
test_mcfost.py::test_SED[ref3.0] PASSED                                                                         [ 18%]
test_mcfost.py::test_mol_map[discF_00500] FAILED                                                                [ 20%]
test_mcfost.py::test_mol_map[ref3.0_multi] FAILED                                                               [ 22%]
test_mcfost.py::test_mol_map[debris] FAILED                                                                     [ 24%]
test_mcfost.py::test_mol_map[ref3.0] PASSED                                                                     [ 26%]
test_mcfost.py::test_image[1.0-discF_00500] FAILED                                                              [ 28%]
test_mcfost.py::test_image[1.0-ref3.0_multi] FAILED                                                             [ 30%]
test_mcfost.py::test_image[1.0-debris] PASSED                                                                   [ 32%]
test_mcfost.py::test_image[1.0-ref3.0] PASSED                                                                   [ 34%]
test_mcfost.py::test_image[10-discF_00500] FAILED                                                               [ 36%]
test_mcfost.py::test_image[10-ref3.0_multi] FAILED                                                              [ 38%]
test_mcfost.py::test_image[10-debris] PASSED                                                                    [ 40%]
test_mcfost.py::test_image[10-ref3.0] PASSED                                                                    [ 42%]
test_mcfost.py::test_image[100-discF_00500] FAILED                                                              [ 44%]
test_mcfost.py::test_image[100-ref3.0_multi] FAILED                                                             [ 46%]
test_mcfost.py::test_image[100-debris] PASSED                                                                   [ 48%]
test_mcfost.py::test_image[100-ref3.0] PASSED                                                                   [ 51%]
test_mcfost.py::test_image[1000-discF_00500] FAILED                                                             [ 53%]
test_mcfost.py::test_image[1000-ref3.0_multi] FAILED                                                            [ 55%]
test_mcfost.py::test_image[1000-debris] FAILED                                                                  [ 57%]
test_mcfost.py::test_image[1000-ref3.0] PASSED                                                                  [ 59%]
test_mcfost.py::test_pola[1.0-discF_00500] SKIPPED (No pola)                                                    [ 61%]
test_mcfost.py::test_pola[1.0-ref3.0_multi] PASSED                                                              [ 63%]
test_mcfost.py::test_pola[1.0-debris] PASSED                                                                    [ 65%]
test_mcfost.py::test_pola[1.0-ref3.0] PASSED                                                                    [ 67%]
test_mcfost.py::test_pola[1000-discF_00500] SKIPPED (No pola)                                                   [ 69%]
test_mcfost.py::test_pola[1000-ref3.0_multi] PASSED                                                             [ 71%]
test_mcfost.py::test_pola[1000-debris] FAILED                                                                   [ 73%]
test_mcfost.py::test_pola[1000-ref3.0] PASSED                                                                   [ 75%]
test_mcfost.py::test_contrib[1.0-discF_00500] SKIPPED (No contrib)                                              [ 77%]
test_mcfost.py::test_contrib[1.0-ref3.0_multi] FAILED                                                           [ 79%]
test_mcfost.py::test_contrib[1.0-debris] FAILED                                                                 [ 81%]
test_mcfost.py::test_contrib[1.0-ref3.0] FAILED                                                                 [ 83%]
test_mcfost.py::test_contrib[100-discF_00500] SKIPPED (No contrib)                                              [ 85%]
test_mcfost.py::test_contrib[100-ref3.0_multi] FAILED                                                           [ 87%]
test_mcfost.py::test_contrib[100-debris] FAILED                                                                 [ 89%]
test_mcfost.py::test_contrib[100-ref3.0] FAILED                                                                 [ 91%]
test_mcfost.py::test_contrib[1000-discF_00500] SKIPPED (No contrib)                                             [ 93%]
test_mcfost.py::test_contrib[1000-ref3.0_multi] FAILED                                                          [ 95%]
test_mcfost.py::test_contrib[1000-debris] FAILED                                                                [ 97%]
test_mcfost.py::test_contrib[1000-ref3.0] FAILED                                                                [100%]
```

**Comment by conradtchan** [comment]
Also, #72 increased the SED test threshold because the test would randomly fail.

**Comment by cpinte** [comment]
Fixed with #95
